### DISCUSSION TITLE
Did not find matching provisioning profiles for code signing!

### DISCUSSION THREAD

**Discussion by C4s4r** [codemagic-ci-cd/codemagic-docs#1132] [Q&A] [answered]
After I successfully created and used a signing certificate and provisioning profile on my MacBook, I wanted to automate this process via Codemagic.
Sadly it always fails the iOS build with the error `Did not find matching provisioning profiles for code signing!`.
Wether I upload my signing certificate and provisioning profile from my MacBook or use the automatic code signing by Codemagic (api working, used correct bundle identifier), the error stays the same.
So I reached the moment where I don't know what mistake I made.

**ID - manual code signing:** 636581cc030e6bfa2b66705c
**ID- automatic code signing:** 6365854f030e6bdf8c7d27e2

### Complete log
```
== Building for iOS ==

> xcode-project use-profiles
Configure code signing settings
Searching for files matching /Users/builder/Library/MobileDevice/Provisioning Profiles/*.mobileprovision
Searching for files matching /Users/builder/Library/MobileDevice/Provisioning Profiles/*.provisionprofile
List available code signing certificates in keychain /Users/builder/Library/codemagic-cli-tools/keychains/04-11-22__wo8agsl.keychain-db
Searching for files matching /Users/builder/clone/ios/**/*.xcodeproj
Completed configuring code signing settings
Did not find matching provisioning profiles for code signing!
Generated options for exporting the project
 - Method: ad-hoc
 - Provisioning Profiles: []
 - Signing Certificate: 
 - Signing Style: manual
 - Team Id: 
Saved export options to /Users/builder/export_options.plist

> flutter build ipa --release --export-options-plist /Users/builder/export_options.plist

💪 Building with sound null safety 💪

Archiving com.example.campusApp...
════════════════════════════════════════════════════════════════════════════════
No valid code signing certificates were found
```

**Comment by himesh-cm** [comment, marked as answer]
Hi, @C4s4r. Can you please check if the bundle identifier is identical in both your project and app store developer account?

**Reply by C4s4r** [reply]
I found the issue.
Everytime I checked for matching bundle identifier, I only checked the Codemagic settings and AppStore Connect id.
The problem was that I had the wrong bundle identifier in my flutter project (in the `project.pbxproj` file).

Thanks for your hint! 👍🏼

**Reply by StoyanTinchev** [reply]
Hey @C4s4r may you tell what exactly is wrong in project.pbxproj because I think that I have the same problem (question: #1290)

**Reply by C4s4r** [reply]
@StoyanTinchev As described above, I had the wrong _bundle identifier_ in my flutter project.
